File: In_C_Programming/q28.c
#include <stdio.h>

void sortinAscending(int arr[], int ln)
{

    for (int i = 0; i < ln - 1; i++)
    {
        for (int j = 0; j < ln - 1 - i; j++)
        {
            if (arr[j] > arr[j + 1])
            {
                arr[j] ^= arr[j + 1];
                arr[j + 1] ^= arr[j];
                arr[j] ^= arr[j + 1];
            }
        }
    }

    printf("Sorted Array in Ascensing Order is: ");

    for (int i = 0; i < ln; i++)
    {
        printf("%d ", arr[i]);
    }

    printf("\n");
}

void sortinDesending(int arr[], int ln)
{

    for (int i = 0; i < ln - 1; i++)
    {
        for (int j = 0; j < ln - 1 - i; j++)
        {
            if (arr[j] < arr[j + 1])
            {
                arr[j] ^= arr[j + 1];
                arr[j + 1] ^= arr[j];
                arr[j] ^= arr[j + 1];
            }
        }
    }

    printf("Sorted Array in Decensing Order is: ");

    for (int i = 0; i < ln; i++)
    {
        printf("%d ", arr[i]);
    }

    printf("\n");
}

void getElement(int arr[], int ln)
{

    printf("Enter The Array Elements; ");

    for (int i = 0; i < ln; i++)
    {
        scanf("%d", &arr[i]);
    }
}

int main()
{

    int ln;

    printf("Enter The Array Length: ");
    scanf("%d", &ln);

    int arr[ln];

    getElement(arr, ln);

    sortinAscending(arr, ln);
    sortinDesending(arr, ln);

    return 0;
}

File: In_Java_Programming/q28.java
import java.util.Scanner;

public class q28 {

    public static void get_Ascending(int arr[], int ln) {

        for (int i = 0; i < ln - 1; i++) {
            for (int j = 0; j < ln - 1 - i; j++) {

                if (arr[j] > arr[j + 1]) {

                    arr[j] ^= arr[j + 1];
                    arr[j + 1] ^= arr[j];
                    arr[j] ^= arr[j + 1];
                }
            }
        }

        System.out.println("The Sorted Array in Ascending Order is: ");

        for (int i = 0; i < ln; i++) {
            System.out.println(arr[i] + " ");
        }
    }

    public static void get_Decending(int arr[], int ln) {

        for (int i = 0; i < ln - 1; i++) {
            for (int j = 0; j < ln - 1 - i; j++) {

                if (arr[j] < arr[j + 1]) {

                    arr[j] ^= arr[j + 1];
                    arr[j + 1] ^= arr[j];
                    arr[j] ^= arr[j + 1];
                }
            }
        }

        System.out.println("The Sorted Array in Decending Order is: ");

        for (int i = 0; i < ln; i++) {
            System.out.println(arr[i] + " ");
        }
    }

    public static void get_Element(int arr[], int ln, Scanner sc) {

        System.out.println("Enter The Array Elements: ");

        for (int i = 0; i < ln; i++) {

            arr[i] = sc.nextInt();
        }
    }

    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        System.out.println("Enter Array Length: ");
        int ln = sc.nextInt();

        int arr[] = new int[ln];

        get_Element(arr, ln, sc);
        get_Ascending(arr, temp, ln);
        get_Decending(arr, ln);

        sc.close();
    }
}